pkg/gits/git_cli_test.go

```diff
@@ -394,3 +394,137 @@ func TestTags(t *testing.T) {
 	assert.Contains(t, tags, "v0.0.1")
 	assert.Contains(t, tags, "v0.0.2")
 }
+
+func TestGitCLI_Stash(t *testing.T) {
```

Comment:
+1 nice with tests. Personally I was hoping to extend the Ginko test suite. IMO there would be several advantages:

* clearer test suite setup and teardown
* better error reports
* ability to focus on tests. 

However, I am just happy to have tests on this level.

Reply:
yeah, tbh I'm not sure why we added Ginko here. I would have preferred to stick with the standard idiom for testing within the project.

Reply:
Fair enough. IMO Ginko has several advantages over the "standard idioms" and hence I prefer using Ginko, especially since it is already used at other places. 

Might be worth having an testing approach discussion across the team at some stage.